stock_dropshipping_dual_invoice/tests/test_need_two_invoices.py

```diff
@@ -73,14 +73,30 @@ def test_so_on_delivery_need_one_invoice(self):
         }).create({})
         self.assertIs(False, wizard._need_two_invoices())
 
+    def test_so_mto_need_one_invoice(self):
+        self.so_mto.order_policy = 'picking'
+        self.so_mto.action_button_confirm()
+        procurements = self.so_mto.procurement_group_id.procurement_ids
+        procurements.run()
+        po = procurements.filtered('purchase_id')[:1].purchase_id
+        self.assertTrue(po)
+        po.invoice_method = 'picking'
+        po.signal_workflow('purchase_confirm')
+        picking = po.picking_ids[:1]
```

Comment:
That's because both pickings are linked to the purchase order, not just the purchase one. I can keep the assert to assure that there's one picking, but I cannot remove this condition. Just one reminder: if `po.picking_ids` is an empty recordset, `po.picking_ids[:1]` will be also an empty recordset